backend/api/views.py

```diff
@@ -189,6 +207,7 @@ def delete_store(self, request, pk=None):
                 )
             user.stores.remove(store)
             store.delete()
+            updateTokenScope(user)
```

Comment:
nice

Reply:
In the delete/remove store, we're essentially updating the user's token to not allow the user to make any authorizations related to that deleted/removed store, right?

Reply:
Yeah, that's the gist of it. Only thing missing here is that it won't remove a deleted store from other user tokens. That might be an issue if someone deletes a store and then someone creates another and they have the same pk (users with access to the old store might be able to access the new store). I'll take a look in a bit to be sure.

Reply:
Hmmm I see. From my experience, I don't believe the store-pks will ever be the same given how the id/pk of the store-model auto increments. Meaning if I delete store with id 1 and some user creates another store, the new store's id would be 2 instead of 1 by the auto-increment behavior. This seems to be a database related thing as said by this [SO  post](https://stackoverflow.com/questions/27735375/django-model-instances-primary-keys-do-not-reset-to-1-after-all-instances-are-de). @MattKersey

Reply:
Just checked - you're totally right about that @ceguiluzrosas. In that case, this should be all set!